app/assets/stylesheets/structure.scss

```diff
@@ -20,6 +22,28 @@
   width: 100%;
 }
 
+.inner-container {
+  @extend .container;
+  width: 1110px;
```

Comment:
Should this be in a media query? Actually I don't know if the app has a responsive layout right now, so maybe it doesn't matter. 🤔